src/index.html

```diff
@@ -2,11 +2,327 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <meta name="viewport"
+      content="width=device-width,
+      initial-scale=1.0"
+    >
+    <title>Uber Eats Pages</title>
     <link rel="stylesheet" href="./styles/main.css">
   </head>
-  <body>
+  <body class="page">
+    <div class="page__wrap">
+      <header class="header">
+        <div class="header__wrap">
+          <a href="#" class="header__logo-link logo">
+            <img src="./images/uber_logo.svg"
+              alt="UberEats logo"
+              class="header__logo-img logo__image"
+            >
+          </a>
+          <form class="location header__loc" action="#">
+            <span class="location__asap--when header__loc--when">When</span>
+            <input type="text"
+              class="location__asap"
```

Comment:
`location_asap = локация быстро как это возможно`?
```suggestion
              class="location__asap"
```